examples/tutorial.py

```diff
@@ -26,21 +26,21 @@ def walktree(top):
         print(child)
 
 # dimensions.
-level = rootgrp.createDimension('level', None)
-time = rootgrp.createDimension('time', None)
-lat = rootgrp.createDimension('lat', 73)
-lon = rootgrp.createDimension('lon', 144)
+level_dim = rootgrp.createDimension('level', None)
+time_dim = rootgrp.createDimension('time', None)
+lat_dim = rootgrp.createDimension('lat', 73)
+lon_dim = rootgrp.createDimension('lon', 144)
```

Comment:
Because the CI now runs mypy (a static type checker) on all examples and tests to find any inconsistencies.
This means that the typing of these files should also be correct.

In this instance, the level variable is a `Dimension` here but was later overwritten with a `Variable`, so the type checker raises an error.

The same happens below with `f`, which sometimes was a file handle, and other times a `Dataset`...